docs/application/tizen-studio/setup/install-sdk.md

```diff
@@ -12,29 +12,27 @@ To install the Tizen Studio:
 
 2. Accept the software license.
 
-   The license contains important legal notices for using the Tizen Studio. Read it fully, and click **Accept** only if you agree with the license statement.
+   The license contains important legal notices for using Tizen Studio. Read it fully, and click **Accept** only if you agree with the  license statement:
```

Comment:
There are two spaces in front of "license". The HTML render remove a space when it generates a HTML file. Nevertheless, you had better remove a space.